framework/branch-key-store.md

```diff
@@ -779,34 +779,10 @@ In addition to the [branch key context](#branch-key-context):
 The Beacon key branch key context MUST includes a key `type` and the value MUST be `"beacon:ACTIVE"`.
 The Beacon key branch key context MUST NOT have a `version` key.
 
-## Encryption Context
-
-This section describes how the AWS KMS encryption context is built.
+## AWS KMS Encryption Context
 
 If the `hierarchy-version` is v1, AWS KMS encryption context MUST be same as [branch key context](#branch-key-context).
-If the `hierarchy-version` is v2, AWS KMS encryption context MUST be the [encryption context](../structures.md#encryption-context) sent by users without any transformation.
-
-If the `hierarchy-version` is v2, the encryption context send to AWS KMS MUST always match the encryption context originally sent by the user, without any transformations. This requirement applies regardless of the type of encryption context being used, whether it is the ACTIVE encryption context, the DECRYPT_ONLY encryption context, the Beacon Key encryption context, or a custom encryption context.
-
-### ACTIVE Encryption Context
-
-If the `hierarchy-version` is v1, ACTIVE Encryption Context is same as [ACTIVE Branch Key Context](#active-branch-key-context)
-
-### DECRYPT_ONLY Encryption Context
-
-If the `hierarchy-version` is v1, DECRYPT_ONLY Encryption Context is same as [DECRYPT_ONLY Branch Key Context](#decrypt_only-branch-key-context)
-
-### Beacon Key Encryption Context
-
-If the `hierarchy-version` is v1, Beacon Encryption Context is same as [Beacon Key Branch Key Context](#beacon-key-branch-key-context)
-
-### Custom Encryption Context
-
-If custom [encryption context](./structures.md#encryption-context)
-is associated with the branch key these values MUST be added to the AWS KMS encryption context.
-To avoid name collisions each added attribute from the custom [encryption context](./structures.md#encryption-context-3)
-MUST be prefixed with `aws-crypto-ec:`.
-Across all versions of a Branch Key, the custom encryption context MUST be equal.
+If the `hierarchy-version` is v2, AWS KMS encryption context MUST be the [encryption context](./structures.md#encryption-context) sent by users without any transformation. This requirement applies regardless of the Branch Key Item's `type`.
```

Comment:
You have this nice section here: #encryption-context-from-authenticated-branch-key-context. It think that this better describes the relationship than "send by the users".